format/Flight.proto

```diff
@@ -259,8 +259,10 @@ message FlightInfo {
   FlightDescriptor flight_descriptor = 2;
 
   /*
-   * A list of endpoints associated with the flight. To consume the whole
-   * flight, all endpoints must be consumed.
+   * A list of endpoints associated with the flight. To consume the
+   * whole flight, all endpoints (and hence all Tickets) must be consumed.
```

Comment:
Is it worth clarifying that the list of endpoints can be consumed in any order?

Reply:
Updated.

Though - that brings up the question of how to represent an ordered dataset. Is the expectation that (say) if results are sorted, then the endpoints should also be sorted?

Reply:
My thought here is that the backend should enforce ordering by grouping ordered data together under one ticket. The client can't process this data in parallel anyway.

This would prevent exposing ordered fragments from separate locations though. It sort of makes sense for a query engine to have to collate data under a single node though.

Reply:
I agree with @jduo on this comment. If data must be ordered, at least for now, it should be exposed via single ticket/endpoint.

Reply:
Pushed a short clarification.